admin/test/test_release.py

```diff
@@ -1874,6 +1874,33 @@ def test_metadata_content_type(self):
             'application/json'
         )
 
+    def test_url_escaped(self):
+        """
+        When a URL includes special characters, they are escaped so that
+        Vagrant can download the box from Amazon S3 without getting 403 errors.
+
+        "/" and ":" are not escaped (these only appear in the protocol).
+        """
+        aws = FakeAWS(
+            routing_rules={},
+            s3_buckets={
+                self.target_bucket: {},
+            },
+        )
+
```

Comment:
not related to this PR, but I found interesting that there is a in-house mock implementation of AWS on ../aws.py.
Just out of curiosity, why wasn't https://github.com/spulec/moto used here ?